scripts/systemd/credentials-fetcher.service

```diff
@@ -3,12 +3,17 @@ Description=credentials-fetcher systemd service unit file.
 
 [Service]
 ExecStartPre=mkdir -p /var/credentials-fetcher/krbdir /var/credentials-fetcher/socket /var/credentials-fetcher/logging
-ExecStartPre=chgrp ec2-user /var/credentials-fetcher /var/credentials-fetcher/krbdir /var/credentials-fetcher/socket /var/credentials-fetcher/logging
-ExecStartPre=chmod 755 /var/credentials-fetcher /var/credentials-fetcher/krbdir /var/credentials-fetcher/socket /var/credentials-fetcher/logging
+ExecStartPre=chgrp ${CF_DIR_GID} /var/credentials-fetcher /var/credentials-fetcher/krbdir /var/credentials-fetcher/socket /var/credentials-fetcher/logging
+ExecStartPre=chmod ${CF_DIR_MODE} /var/credentials-fetcher /var/credentials-fetcher/krbdir /var/credentials-fetcher/socket /var/credentials-fetcher/logging
 ExecStart=/usr/sbin/credentials-fetcherd
-ExecStartPost=chgrp ec2-user /var/credentials-fetcher/socket/credentials_fetcher.sock
-ExecStartPost=chmod 660 /var/credentials-fetcher/socket/credentials_fetcher.sock
 Environment="CREDENTIALS_FETCHERD_STARTED_BY_SYSTEMD=1"
+Environment="CF_CRED_SPEC_FILE=/var/credentials-fetcher/credspec.json"
+Environment="CF_DIR_GID=1000"
+Environment="CF_DIR_MODE=755"
+Environment="CF_KRB5CC_GID=1000"
```

Comment:
TBD: CF_KRB5CC_GID can be ec2-user after OS check

Reply:
GID=1000 is the numeric value of ec2-user